test/unit/slurm/test_slurm_config_generator/test_generate_slurm_config_files_slurm_accounting/expected_outputs/slurm_parallelcluster_external_slurmdbd.conf

```diff
@@ -0,0 +1,18 @@
+# slurm_parallelcluster.conf is managed by the pcluster processes.
+# Do not modify.
+# Please use CustomSlurmSettings in the ParallelCluster configuration file to add user-specific slurm configuration
+# options
+
+SlurmctldHost=ip-1-0-0-0(ip.1.0.0.0)
+SuspendTime=600
+ResumeTimeout=1600
+SelectTypeParameters=CR_CPU
+AccountingStorageType=accounting_storage/slurmdbd
+AccountingStorageHost=test.slurmdbd.host
+AccountingStoragePort=6819
+AccountingStorageUser=slurm
+JobAcctGatherType=jobacct_gather/cgroup
+
+include <DIR>/pcluster/slurm_parallelcluster_efa_partition.conf
```

Comment:
Why we need to include efa configuration in the slurm_dbd.conf ?

Reply:
This is not the `slurmdbd.conf`, this is still the `slurm.conf`. But I see that the name of the file can be misleading (here I'm using `external_slurmdbd` as an appendix for the files in the external dbd case).
What appendix should we use here? `externaldbd`?

Reply:
Ahh.. ok. Yes, that is misleading.
Let's try with "_externaldbd"

Reply:
Done. Resolving.